### PR TITLE
add call_trigger variable

### PR DESCRIPTION
**Pull request by atm-greg**
# FIX missing $call_trigger variable

### CODE REVIEW

**Comment by atm-maxime** [comment]
Have a look in the modulebuilder myobject.class.php, you'll see the standard variable used is "no trigger" set to false by default. 😉

**Comment by atm-greg** [comment]
@atm-maxime I have done this like that because i found this variable on the update method of societe.class.php.

Then i realized that $call_trigger = 1 was more understandable than !$notrigger without changing the basic behaviour.
And the parameter wasn't present before.

File: htdocs/contact/class/contact.class.php [hunk 1]
```diff
@@ -432,10 +432,11 @@ public function load_state_board()
...
 	 *  Add a contact into database
 	 *
-	 *  @param      User	$user       Object user that create
-	 *  @return     int      			<0 if KO, >0 if OK
+	 *  @param      User	$user           Object user that create
+     *  @param      int     $call_trigger   0=no, 1=yes
+	 *  @return     int      			    <0 if KO, >0 if OK
 	 */
-	public function create($user)
+	public function create($user, $call_trigger = 1)
```
Comment (eldy) [inline htdocs/contact/class/contact.class.php:439]: To follow standard, using $notrigger = 0 would be a better choice.

**Comment by atm-greg** [comment]
@eldy fixed

**Pull request merged**